cmd/install/assets/hypershift_operator.go

```diff
@@ -237,6 +238,12 @@ func (o HyperShiftOperatorDeployment) Build() *appsv1.Deployment {
 									Protocol:      corev1.ProtocolTCP,
 								},
 							},
+							Resources: corev1.ResourceRequirements{
+								Requests: corev1.ResourceList{
+									corev1.ResourceMemory: resource.MustParse("157286400"),
```

Comment:
what's this value coming from?

Reply:
They were the corresponding values for 150Mi and 10m as integers but I've normalized them below to make them more readable